mirascope/cli/cli.py

```diff
@@ -54,7 +54,8 @@ def main():
 
     # Adding 'init' command
     parser_init = subparsers.add_parser("init", help="Initialize mirascope project")
```

Comment:
should we make the help `Initialize Mirascope project` i.e. capitalize, here and everywhere else?

Reply:
in the context of library everything is lowercase i think?

Reply:
@wcbudz what are your thoughts here?

Reply:
I think when referring specifically to the library it's fine for lowercase because that's how people will type it in code.